give_kudos.py

```diff
@@ -102,9 +107,23 @@ def locate_kudos_buttons_and_maybe_give_kudos(self, web_feed_entry_locator) -> i
                 if not self.is_participant_me(web_feed):
                     button = self.find_unfilled_kudos_button(web_feed)
                     given_count += self.click_kudos_button(unfilled_kudos_container=button)
-        print(f"Kudos given: {given_count}")
+        print(f"\nKudos given: {given_count}")
         return given_count
     
+    def is_club_post(self, container) -> bool:
+        """
+        Returns true if the container is a club post
+        """
+        print("group header:", container.get_by_test_id("group-header").count())
+        if(container.get_by_test_id("group-header").count() > 0):
+            return True
+        
+        print("club member post:", container.locator(".clubMemberPostHeaderLinks").count())
+        if(container.locator(".clubMemberPostHeaderLinks").count() > 0):

```

Comment:
Remove these print statements?

Reply:
good catch! I removed other debug lines, but these two slipped trough 😄

Reply:
yes, need to remove these two print statements